packages/moveable/src/moveable.tsx

```diff
@@ -28,13 +29,20 @@ import InOverlay from './overlay';
 
 const DEFAULT_Z_INDEX = 10;
 
-function MoveableWithRef({ onContextMenu, ...moveableProps }, ref) {
+interface MoveableWithRefProps extends MoveableProps {
+  onContextMenu: MouseEventHandler<HTMLElement>;
+}
+
+function MoveableWithRef(
+  { onContextMenu, ...moveableProps }: MoveableWithRefProps,
+  ref: ForwardedRef<OriginalMoveable>
```

Comment:
I think this fixes it. For some reason, Moveable breaks with regular practices and makes the `ref` a reference to the component itself and not to either an HTML element or some imperative handle. It's probably because it's still written class-based and not converted to hooks.

Reply:
Thanks for looking into it!